Docs/Components-pages/Badges-page.html

```diff
@@ -0,0 +1,111 @@
+        </aside>
+        <section class="content-section">
+            <section class="content-subsection flex">
+                <p class="heading"> Badges </p>
+                <p class="content">
+                    Badges are used to show user status and sometimes notification counts.
+                </p>
+                <section class="content-subsection subsection flex">
+                    <h2 class="component-demo-heading"> Badges on avatar </h2>
+                    <p class="content"> Badges on avatars usually show user is active or not. To use badges on avatar in your project add below html code to your project or just add classes to your elements.</p>
+                    <div class="avatar">
+                        <div class="flex">
+                            <div class="round-avatar-img">
+                                <img src="/components/images/avatar-img.jpeg" alt="Guy wearing a hoodie"
+                                    class="md-rounded-avatar" />
+                            </div>
+                            <div class="red-badge avatar-badge"></div>
+                        </div>
+                        <div class="flex">
+                            <div class="round-avatar-img">
+                                <img src="/components/images/avatar-img.jpeg" alt="Guy wearing a hoodie"
+                                    class="md-rounded-avatar" />
+                            </div>
+                            <div class="green-badge avatar-badge"></div>
+
+                        </div>
+                        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C--inactive%2520badge--%253E%250A%250A%253Cdiv%2520class%253D%2522flex%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522round-avatar-img%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522img-path.jpg%2522%2520alt%253D%2522alt-text%2522%2520%250A%2520%2520%2520%2520%2520%2520%2520class%253D%2522md-rounded-avatar%2522%2520%252F%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522red-badge%2520avatar-badge%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253C--active%2520badge%2520--%253E%250A%250A%253Cdiv%2520class%253D%2522flex%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522round-avatar-img%2522%253E%250A%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522img-path.jpg%2522%2520alt%253D%2522alt-text%2522%250A%2520%2520%2520%2520%2520%2520class%253D%2522md-rounded-avatar%2522%2520%252F%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522green-badge%2520avatar-badge%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 538px; height: 540px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
```

Comment:
instead of `px` use `rem` units.

Reply:
These styles were outsourced with ifram for code image. So I didn't change anything.